doc/nrf/protocols/thread/configuring.rst

```diff
@@ -120,12 +121,16 @@ By enabling support for Thread 1.2, you enable the following :ref:`thread_ug_sup
 * Enhanced Frame Pending
 * Enhanced Keep Alive
 
-By selecting support for Thread 1.3, you enable the following features in addition to the :ref:`Thread 1.2 features <thread_ug_supported_features>`:
+By selecting support for Thread 1.3, you enable the following :ref:`thread_ug_supported_features_v13` in addition to the :ref:`thread_ug_supported_features_v12`:
 
 * Service Registration Protocol (SRP) client
 
+By selecting support for Thread 1.4, you enable the following :ref:`thread_ug_supported_features_v14` in addition to the :ref:`thread_ug_supported_features_v14`:
+
+* Enhanced Internet Connectivity
+* Enhanced Network Diagnostics
```

Comment:
List does not match the list on the features page. Same is true for 1.3.

Reply:
That's because by selecting we enable only part of `CONFIG_OPENTHREAD_THREAD_VERSION_1_X`  only part of features (listed here) is enabled by default. the ones not listed here have additional instructions on how to enable them in `supported_features.rst`, like for TCAT:

```
See the :file:`tcat.conf` configuration file in the :file:`snippets/tcat/` directory of the :ref:`ot_cli_sample` sample for an example how to enable the TCAT functionality.
```